### DISCUSSION TITLE
[20240127] Can't backup setting

### DISCUSSION THREAD

**Discussion by kw-sudo** [rern/rAudio#1583] [Fixed Issues] [answered]
Hi rern,

I've just updated to version 20240127. If I press the settings backup  button, it keeps (highlighted blue) in "active" position but no download option appears.

`rAudio 20240127
6.6.13-2-rpi aarch64
Raspberry Pi 3B Rev 1.2`

Edit: PS: restore option works fine

Regards,
Oliver

**Comment by klassikmann** [comment]
Confirmed.

**Comment by kw-sudo** [comment, marked as answer]
@rern 

With v202402010 it's fixed. Thank you!

Regards,
Oliver
